content/en/docs/concepts/security/index.md

```diff
@@ -755,7 +754,9 @@ spec:
 #### `allow-nothing`, `deny-all` and `allow-all` policy
 
 The following example shows an `ALLOW` policy that matches nothing. If there are no other `ALLOW` policies, requests
-will always be denied because of the "deny by default" behavior.
+will always be denied because of the "deny by default" behavior when there are `ALLOW` policies applied but none matched.
+
+Note the "deny by default" behavior applies only to the `ALLOW` policy and not other policies.
```

Comment:
Apologies, I don't know GitHub that well. Anyway, please change the allow-nothing example to the following:

```
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: allow-nothing
spec:
  action: ALLOW
  {}
```

The existing comment really threw me off. I didn't realize that it was talking about the action field. I think it's better to simply add the action here since we say it's a best practice to always include the action.

Reply:
I updated the example and its comments, it is for the `rules` field.